test/Release/Gateway/ReleaseToggleCollectionMock.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace Clearbooks\Labs\Release\Gateway;
+
+use Clearbooks\Labs\Toggle\Entity\Toggle;
+
+class ReleaseToggleCollectionMock implements ReleaseToggleCollection
+{
+    /** @var string */
+    public $releaseId;
+
+    /**
+     * @param string $releaseId
+     * @return Toggle[]
+     */
+    public function getTogglesForRelease( $releaseId )
+    {
+        $this->releaseId = $releaseId;
```

Comment:
Is that a thing in PHP? I know it is in Java, but would it matter anything at all as a PHP annotation?